hud-data/toucahud.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "toucaHUD",
+  "author": "touca",
+  "contributors": [
+    "Jofre",
+    "Hypnotize",
+    "Colly",
+    "Omnibombulator"
+  ],
+  "releaseDate": "2024-10-03",
+  "parent": "hexhud, collyhud",
+  "distinguishedFromParent": true,
+  "resources": [
+    "https://imgur.com/2CbDLfM"
+  ],
+  "flags": [
+    "fonts",
+    "menus",
+    "scoreboards",
+    "minmode",
+    "materials"
+  ],
+  "tags": [
+    "Minimal",
```

Comment:
Please lower-case tags.

Reply:
made all the changes, lemme know if there's anything else you need me to do